src/packaging_tool_versions.rs

```diff
@@ -0,0 +1,20 @@
+use serde::{Deserialize, Serialize};
+
+/// The versions of various packaging tools used during the build.
+/// These are always installed, and are independent of the chosen package manager.
+#[derive(Clone, Deserialize, PartialEq, Serialize)]
+pub(crate) struct PackagingToolVersions {
+    pub pip_version: String,
+    pub setuptools_version: String,
+    pub wheel_version: String,
```

Comment:
Question: Are there better ways of representing these versions than `String`? For example, if they're SemVer we could leverage the `semver` crate. However, if all we do in this buildpack is comparing them verbatim `String` seems fine.

Reply:
These versions don't use semver (for example, Pip doesn't use `X.Y.0` but `X.Y` for the first release of a new minor). They are only used as strings, and also the version handling here will change as part of [Reduce toil required to keep pip/setuptools/wheel up to date](https://gus.lightning.force.com/lightning/r/ADM_Epic__c/a3QEE000000WdUT2A0/view) (which will switch to pulling the values from requirements files, so Dependabot can keep them up to date).